OpenRA.Mods.RA/Traits/PortableChrono.cs

```diff
@@ -40,17 +40,19 @@ class PortableChronoInfo : ITraitInfo
 
 		[VoiceReference] public readonly string Voice = "Action";
 
-		public object Create(ActorInitializer init) { return new PortableChrono(this); }
+		public object Create(ActorInitializer init) { return new PortableChrono(this, init.Self); }
 	}
 
 	class PortableChrono : IIssueOrder, IResolveOrder, ITick, ISelectionBar, IOrderVoice, ISync
 	{
 		[Sync] int chargeTick = 0;
 		public readonly PortableChronoInfo Info;
+		public readonly Actor Self;
```

Comment:
This shouldn't need to be public, so use `readonly Actor self;` and `this.self = self` in the constructor.